test/unit/test_utils.js

```diff
@@ -15,6 +15,14 @@
 
 import { CMapCompressionType } from '../../src/shared/util';
 
+class NodeFileReader {
+  constructor(params) {
+    var fs = require('fs');
+    var file = fs.readFileSync(params.path);
+    return new Uint8Array(file);
```

Comment:
It looks like it is indeed valid ES6 syntax: https://stackoverflow.com/questions/27980996/return-a-value-other-than-the-class-in-es6

I'm thinking in that case it may be good to add a small comment on why we make the constructor return a different object.